client/src/components/shop/ProductCard.js

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useReducer, useState } from "react";
```

Comment:
where are you using reducer and state here?

Reply:
Good point. Must have been left there by accident when I was splitting up code.